qtaf-html-report-plugin/src/main/java/de/qytera/qtaf/htmlreport/event_subscriber/CreateReportSubscriber.java

```diff
@@ -22,6 +24,13 @@ public class CreateReportSubscriber implements IEventSubscriber {
 
     @Override
     public void initialize() {
+        ConfigMap config = ConfigurationFactory.getInstance();
+        boolean htmlReportIsEnabled = config.getBoolean("htmlReport.enabled");
```

Comment:
Just add "false" as second argument to "getBoolean". This will set the value to false if no value was found in the configuration file.

Reply:
The requested change is pushed